components/channel/publish.js

```diff
@@ -27,13 +27,14 @@ function getPublishCode(publishMessage, realizedChannelPath) {
  * @param {*} channelName 
  * @param {*} channelParameters 
  * @param {*} publishMessage 
+ * @param {*} params 
  */
-export default function publish(channelName, channelParameters, publishMessage) {
+export default function publish(channelName, channelParameters, publishMessage, params) {

```

Comment:
I don't see any place where you use `params` argument in this function.

Reply:
Yep, removed 😄